examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -100,6 +101,9 @@ CHIP_ERROR CHIPCommand::MaybeSetUpStack()
     ReturnLogErrorOnFailure(mOperationalKeystore.Init(&mDefaultStorage));
     ReturnLogErrorOnFailure(mOpCertStore.Init(&mDefaultStorage));
 
+    // Initialized with a non-persistent keystore without PSA(chip-tool has not yet build with PSA)
```

Comment:
This is not documenting _why_ a non-persistent keystore here is ok.

Reply:
documented. thanks